packages/adminrouter/extra/apache-windows/adminrouter.conf

```diff
@@ -0,0 +1,72 @@
+
+ServerRoot "c:\opt\mesosphere\bin\Apache24"
+
+Listen 61001
+
+LoadModule access_compat_module modules/mod_access_compat.so
+LoadModule auth_basic_module modules/mod_auth_basic.so
+LoadModule authz_core_module modules/mod_authz_core.so
+LoadModule log_config_module modules/mod_log_config.so
+LoadModule proxy_module modules/mod_proxy.so
+LoadModule proxy_http_module modules/mod_proxy_http.so
+
+DocumentRoot "c:\opt\mesosphere\bin\Apache24/htdocs"
+
+<Directory />
+    AllowOverride none
+    Require all denied
+</Directory>
+
+# Configures the footer on server-generated documents
+ServerSignature Off
+
+# Configures to send just "Server:Apache" in the Server HTTP response header 
+ServerTokens Prod
+
+<VirtualHost *:61001>
+    # Enforce 1024M request body max size limit for DC/OS 
+    LimitRequestBody 1073741824
+    # Set keepalive connections timeout for DC/OS
+    KeepAliveTimeout 65
+    # Settings for Apache proxy options
+    ProxyPreserveHost On
+    # Disable forward proxy
+    ProxyRequests Off
+    <Proxy *>
+        Order allow,deny
+        Allow from all
+    </Proxy>
+
+    # Proxy routing for DC/OS Metrics service
+    <Location /system/v1/metrics>
+        ProxyPass        http://localhost:9000
+        ProxyPassReverse http://localhost:9000
+    </Location>
+
+    # Proxy routing for DC/OS Dignostics service
+    <Location /system/health/v1>
+        ProxyPass        http://localhost:9003/system/health/v1
+        ProxyPassReverse http://localhost:9003/system/health/v1
+    </Location>
+    # Proxy routing for DC/OS Pkgpanda service
+    <Location /pkgpanda>
+        ProxyPass        http://localhost:9001
+        ProxyPassReverse http://localhost:9001 
+    </Location>
+    # Routing for DC/OS Logging service
+    <Location /system/v1/logs>
+        ProxyPass        http://localhost:9002
+        ProxyPassReverse http://localhost:9002
+    </Location>
+
+</VirtualHost>
+
+ErrorLog "logs/error.log"
+LogLevel warn
+
+CustomLog "logs/access.log" common
+<IfModule log_config_module>
```

Comment:
`log_config_module` should be always included, let's please remove the `IfModule` block.

Reply:
Good catch. addressed